script.js

```diff
@@ -1 +1,186 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+// 전역
+const todoUl = document.querySelector(".todo-box__todo ul");
+const doneUl = document.querySelector(".todo-box__done ul");
+// Todo, Done 리스트를 저장하는 전역 변수
+let todoArr = [];
+let doneArr = [];
+
+// 오늘 날짜 반환 함수
+const getTodayDate = () => {
+  const today = new Date();
+  const week = ["일", "월", "화", "수", "목", "금", "토"];
+
+  const year = today.getFullYear();
+  const month = ("0" + (today.getMonth() + 1)).slice(-2);
+  const day = ("0" + today.getDate()).slice(-2);
+  const dayOfWeek = week[today.getDay()];
+
+  https: return `${year}.${month}.${day} ${dayOfWeek}요일`;
+};
+
+// Local Storage에 저장
+const setLocalStorage = () => {
+  localStorage.setItem("todo", JSON.stringify(todoArr));
+  localStorage.setItem("done", JSON.stringify(doneArr));
+};
+
+// Local Storage 불러오기
+const loadLocalStorage = () => {
+  const todo = JSON.parse(localStorage.getItem("todo"));
+  const done = JSON.parse(localStorage.getItem("done"));
```

Comment:
localStorage가 비어있으면 null을 반환해서 이후에 forEach를 돌면 아래와 같이 오류가 나는 것 같아서 예외처리해주면 좋을 것 같습니다!
![image](https://github.com/CEOS-Developers/vanilla-todo-19th/assets/15906101/300e60dd-a613-415d-a1a0-bfcca6c4546c)
```suggestion
  const todo = JSON.parse(localStorage.getItem("todo")) || [];
  const done = JSON.parse(localStorage.getItem("done")) || [];
```